content/rdi/architecture.md

```diff
@@ -9,59 +9,89 @@ headerRange: "[2]"
+
+Redis Data Integration currently supports two use cases:
+
+- Ingest (cache prefetch).
+- Write Behind.
+
+## Ingest functionality and architecture
+
+You can think of RDI Ingest as a streaming ELT process:
+
+- Data is **E**xtracted from the source database using [Debezium Server](https://debezium.io/) connectors.
 - Data is then **L**oaded into Redis Data Integration, a Redis database instance that keeps the data in [Redis streams](https://redis.io/docs/manual/data-types/streams/) alongside required metadata.
 - Data is then **T**ransformed using provided [RedisGears recipes](https://developer.redis.com/howtos/redisgears/) and written to the target Redis DB.
 
-Redis Data Integration using Debezium works in 2 modes:
+Redis Data Integration using Debezium Server works in two modes:
 
 1. Initial sync - Where a snapshot of the entire database or a subset of selected tables is used as a baseline. The entire dataset is streamed to Redis Data Integration and then transformed and written into the target Redis database.
 2. Live updates - Where Debezium captures changes to the data that happen after the baseline snapshot and streams them to Redis Data Integration where they are transformed and written to the target.
 
 ![High-level Redis Data Integration architecture](/images/rdi/redis-di-simplified.png)
 
-## Supported data transformations
+### Supported data transformations
+
+#### Mode mapping
+
+RDI supports conversion of a database record into the following Redis types:
+
+- Hash
+- Set
+- Sorted Set
+- Stream
+- JSON
+- String
+
+Each column in the record is automatically formatted as a number or string. See the [data types list]({{<relref "/rdi/reference/data-types-conversion">}}) for the exact mappings.
+
+#### Additional transformations
 
-Currently Redis Data Integration supports transforming data from RDBMS row into a [Redis Hash](https://redis.io/docs/manual/data-types/#hashes).
-In the short term we will add support for RDBMS row into [RedisJSON](https://redis.io/docs/stack/json/) document.
-See the [data types list]({{<relref "/rdi/reference/data-types-conversion">}}) for the exact mappings.
+RDI supports declarative transformations to further manipulate the data, including denormalization of data from several records into a single JSON key in Redis.
 
-## Architecture and components
+Read the data transformation section of these docs to learn more.
+
+### Architecture and components
 
 ![Detailed Redis Data Integration architecture](/images/rdi/redis-di.png)
 
-## Feeders
+#### Feeders
 
-Currently the only source for Redis Data Integration is the [Debezium Redis Sink Connector](https://debezium.io/documentation/reference/stable/operations/debezium-server.html#_redis_stream).
-Other sources and SDK will be added in later versions.
+Debezium Server has a Redis sink that writes data and state to RDI streams.
+In addition RDI works with [Arcion](arcion.io), a commercial CDC that provides additional set of sources.
```

Comment:
Consider adding an empty line above or they won't be separated.

Reply:
done